java-frontend/src/main/java/org/sonar/java/se/checks/OptionalGetBeforeIsPresentCheck.java

```diff
@@ -107,7 +108,9 @@ public void visitMethodInvocation(MethodInvocationTree tree) {
       if (OPTIONAL_IS_PRESENT.matches(tree)) {
         constraintManager.setValueFactory(id -> new OptionalSymbolicValue(id, programState.peekValue()));
       } else if (OPTIONAL_GET.matches(tree) && presenceHasNotBeenChecked(programState.peekValue())) {
-        context.reportIssue(tree, check, "Call \"" + getIdentifierPart(tree.methodSelect()) + "isPresent()\" before accessing the value.");
+        String identifier = getIdentifierPart(tree.methodSelect());
+        context.reportIssue(tree, check, String.format("Call \"%sisPresent()\" before accessing the value.", StringUtils.isEmpty(identifier) ? "Optional#" : (identifier + ".")),
```

Comment:
I would avoid having two conditional operators testing the same thing here : a small if filling properly some well named variable should do. 
And I tend to think String format on this very case is a bit of overkill, concatenation would do the trick nicely.